modules/nodes-descheduler-profiles.adoc

```diff
@@ -90,9 +90,69 @@ Do not enable `CompactAndScale` with any of the following profiles: `LifecycleAn
 
 endif::nodes[]
 ifdef::virt[]
-Use the `LongLifecycle` profile to enable the descheduler on a virtual machine. This is the only descheduler profile currently available for {VirtProductName}. To ensure proper scheduling, create VMs with CPU and memory requests for the expected load.
+Use the `DevKubeVirtRelieveAndMigrate` or `LongLifecycle` profile to enable the descheduler on a virtual machine.
+
+[IMPORTANT]
+====
+You can not have both `DevKubeVirtRelieveAndMigrate` and `LongLifeCycle` enabled at the same time.
+====
 endif::virt[]
 
+`DevKubeVirtRelieveAndMigrate`:: This profile is an enhanced version of the `LongLifeCycle` profile. It evicts pods from high-cost nodes to reduce overall resource expenses and enable workload migration. Node costs can include the following:
```

Comment:
I think it's also worth to mention that periodically rebalancing the cluster is also ensuring we have almost the  same spare capacity on all nodes in order to prepare for the worst case (on any node, there is a similar amount of spare capacity to deal with spikes).